lib/commands/utility/ping.js

```diff
@@ -1,4 +1,4 @@
-// giveRole.js
+// ping.js
```

Comment:
this comment error is likely also in every other command file. I may have not updated it after copy and pasting...

Reply:
Yup lmao, ill go through and fix them eventually